Extension/ui/settings.html

```diff
@@ -667,6 +667,15 @@
             </div>
         </div>
 
+        <div class="section">
+            <div class="section-title" data-loc-id="merge.configurations">Merge configurations</div>
+            <div>
+                <input type="checkbox" id="mergeConfigurations" style="vertical-align: middle; transform: scale(1.5)">
+                    <span data-loc-id="merge.configurations.description">Set to `true` to merge include paths, defines, and forced includes with those from a configuration provider.</span>

```

Comment:
Since this is a checkbox and not an input value, we'll have to change the direction that describes checking the box:

"When true (or checked), merge include paths, defines, and forced includes with those from a configuration provider."

Reply:
Good call